server/record/recordManager.js

```diff
@@ -105,12 +108,21 @@ const updateRecordStep = async (surveyId, recordUuid, stepId) => {
  * DELETE
  * ===================
  */
-const deleteRecord = async (user, surveyId, recordUuid) =>
+const deleteRecord = async (user, surveyId, recordUuid) => {
   await db.tx(async t => {
     await ActivityLog.log(user, surveyId, ActivityLog.type.recordDelete, { recordUuid }, t)
     await RecordRepository.deleteRecord(user, surveyId, recordUuid, t)
   })
 
+  const recordUsersIds = RecordUpdateManager.getRecordUsers(recordUuid)
```

Comment:
I would suggest to put this logic into RecordUpdateManager. it can have a method like notifyUsersRecordDeleted and leave the logic of the web socket in the RecordUpdateManager